code/dataobjects/Block.php

```diff
@@ -428,6 +428,14 @@ public function CSSClasses($stopAtClass = 'DataObject')
     }
 
     /**
+     * Access current page scope from Block templates with $CurrentPage
+     * @return SiteTree | Controller
+     */
+    public function getCurrentPage() {
+        return Director::get_current_page();
```

Comment:
I used `getPageController`, not sure what's best. I see you already discussed this in https://github.com/sheadawson/silverstripe-blocks/issues/69, so I'm happy with any name you guys come up with.